app/src/main/java/com/devindi/wallpaper/model/map/TileUtils.kt

```diff
@@ -0,0 +1,50 @@
+package com.devindi.wallpaper.model.map
+
+import com.devindi.wallpaper.misc.RectD
+import org.osmdroid.tileprovider.cachemanager.CacheManager
+import timber.log.Timber
+
+class TileUtils {
+
+    fun getTilesRect(centerLat: Double, centerLon: Double, width: Int, height: Int, tileSize: Int, zoom: Int): RectD {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/devindi/mapwall/blob/63cb55c913b0d690492f74a58d6634a0c41df234/app/src/main/java/com/devindi/wallpaper/model/map/TileUtils.kt#L9"></span>Parameter should be on a separate line (unless all parameters can fit a single line)</td>
    </tr>
  </tbody>
</table>